introduction.md

```diff
@@ -0,0 +1,4 @@
+Hi. I ma restarting the course because your valdiation scripts are flaky.  Thanks for making it harder than it should by requiring me to write soem crap.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.